node/tests/st/bgp/test_switch_route_reflector.py

```diff
@@ -1,145 +0,0 @@
-# Copyright (c) 2021 Tigera, Inc. All rights reserved.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-import logging
-import yaml
-
-from nose.plugins.attrib import attr
-from multiprocessing.dummy import Pool as ThreadPool
-from unittest import skip
-
-from tests.st.test_base import TestBase
-from tests.st.utils.docker_host import DockerHost, CLUSTER_STORE_DOCKER_OPTIONS
-from tests.st.utils.route_reflector import RouteReflectorCluster
-from tests.st.utils.utils import (
-    check_bird_status,
-    retry_until_success,
-    update_bgp_config
-)
-
-from .peer import create_bgp_peer
-
-logger = logging.getLogger(__name__)
-
-class TestSwitchRouteReflector(TestBase):
-
-    @attr('slow')
-    def _test_switch_route_reflector(self, backend='bird', bgpconfig_as_num=64514, peer_as_num=64514):
-        """
-        Test that switching from node-to-node full mesh to route reflectors doesn't disrupt dataplane traffic if done as per
-        https://projectcalico.docs.tigera.io/networking/bgp#change-from-node-to-node-mesh-to-route-reflectors-without-any-traffic-disruption
```

Comment:
Need replacement for this one?  Seems to require more than the current e2e connchecker can do.

Reply:
Yeah, the continuous traffic checking is beyond what it can do - I was intending to add an enhancement for continuous checking in a follow-on (maybe using iperf, or just a looping ping of some sort).

I'll see about just adding it to this batch, though.

Reply:
https://tigera.atlassian.net/browse/CORE-12003

I also have a follow-on PR in the works but think it is large enough to warrant its own PR.